### DISCUSSION TITLE
Issue with Naruto X Boruto Ultimate Ninja Storm Connections

### DISCUSSION THREAD

**Discussion by Necrow51** [nefarius/BthPS3#94] [Support]
Hello
I use DS3 controller with BthPS3 Xinput mode, work like a charm with all of my game but except with the game mentioned in the title.
Trigger buttons L2 & R2 don't work (supports & guard) the keys don't respond, same thing if I want to remap in the game options...
Any fix? Or I need to wait a game updated? :(

**Comment by nefarius** [comment]
BthPS3 + DsHidMini offers exactly what's documented. Every button and axis works and is tested. We can't fix individual game's troubles.

**Reply by Necrow51** [reply]
I don't doubt it, as I said everything works fine except with this game so it's independent of you, I just wanted to be sure...
Thanks
